fontir/src/coords.rs

```diff
@@ -161,6 +167,32 @@ impl NormalizedCoord {
     }
 }
 
+impl<T> Location<T> {
+    pub fn new(positions: BTreeMap<String, T>) -> Location<T> {
```

Comment:
instead of requiring this to be a specific type, you could make this constructor take like `impl IntoIterator<Item=(String, T)>` or similar, and we can keep the BTreeMap as an implementation detail.

Reply:
(and you could even get fancy and implement `FromIterator` for location, which would mean you would work with `collect()`, directly)

Reply:
Oh good call, will update. Definitely not comfortable with some of those nuances yet.